modules/sdp/LocalSdpMunger.js

```diff
@@ -205,15 +206,29 @@ export default class LocalSdpMunger {
                 if (ssrcLine.value) {
                     const streamAndTrackIDs = ssrcLine.value.split(' ');
 
-                    if (streamAndTrackIDs.length === 2) {
-                        ssrcLine.value
-                            = this._generateMsidAttribute(
-                                mediaSection.mLine?.type,
-                                streamAndTrackIDs[1],
-                                streamAndTrackIDs[0]);
-                    } else {
-                        logger.warn(`Unable to munge local MSID - weird format detected: ${ssrcLine.value}`);
+                    let streamId = streamAndTrackIDs[0];
```

Comment:
`streamId` seems to be used only in the feature flagged conditional, maybe it can be moved inside the `if` body?